hub/@jmackie/quote.json

```diff
@@ -0,0 +1,7 @@
+{
+  "workspace": "@jmackie/block-quote",
```

Comment:
Could you remove this line please?

We use this for cases where the block source is inside a [yarn workspace](https://classic.yarnpkg.com/lang/en/docs/workspaces/), which isn't the case for your block.

Reply:
Thanks. Sorry about that.